packages/create-cloudflare/templates/hello-world-python/c3.ts

```diff
@@ -1,9 +0,0 @@
-export default {
-	configVersion: 1,
-	id: "hello-world-python",
-	displayName: '"Hello World" Worker (Python)',
-	platform: "workers",
-	copyFiles: {
-		path: "./py",
-	},
-};
```

Comment:
Is this deletion intentional?

Reply:
Yes. The hello-world-python template is merged into the hello-world template.

Reply:
Maybe I'm missing something but only the config file has been deleted – should the whole template be deleted?

Reply:
They are moved inside the `hello-world` folder with the `python` variant [added to the config](https://github.com/cloudflare/workers-sdk/pull/6320/files#diff-68d52b4c5d74735efdc8dcfe9606b81934e4f0f7646e7acac09b3cd0c3455dccR14-R16).